client/src/app/site/base/base-view-model.ts

```diff
@@ -9,6 +9,7 @@ import { BaseModel } from '../../domain/models/base/base-model';
 
 export interface ViewModelConstructor<T extends BaseViewModel> {
     COLLECTION: string;
+    DEFAULT_FIELDSET?: string[];
```

Comment:
I don't think you need this in both the models AND view-models

Reply:
I removed it from ViewModel but I still need this for models. I think the naming might be misleading as this property should just contain all fields one can request from autoupdate.